### DISCUSSION TITLE
Get text: Get last command

### DISCUSSION THREAD

**Discussion by ph0nky-town** [kovidgoyal/kitty#7578] [Q&A]
Hi there,

I scrolled through the documentation, but didn't find how to get the last command from the kitty terminal.
Get text only allows to recover command outputs, but is there any way to also get the command that was entered by the user?

Thanks a lot for your help !

**Comment by kovidgoyal** [comment]
No, there isnt, though why would you need that? You can get the last
command directly from your shell.

**Reply by ph0nky-town** [reply]
I'm toying around with LLMs and I was trying get the last command + output so that I can send it to OpenAI directly from within kitty and ask it to explain the output for example. But ideally I would send the command too to give it some context on the output.

**Reply by kovidgoyal** [reply]
get the last command from the shell using "history" or whatever such facility your shell offers and get the last output from kitty since the shell cant give you that.

**Reply by ph0nky-town** [reply]
Alright I'll tinker with it, thanks !
